docs/docs/09-tooling-integration.md

```diff
@@ -32,11 +32,41 @@ If you like using JSX, we provide an in-browser JSX transformer for development
 
 ### Productionizing: Precompiled JSX
 
-If you have [npm](https://www.npmjs.com/), you can simply run `npm install -g react-tools` to install our command-line `jsx` tool. This tool will translate files that use JSX syntax to plain JavaScript files that can run directly in the browser. It will also watch directories for you and automatically transform files when they are changed; for example: `jsx --watch src/ build/`.
+If you have [npm](https://www.npmjs.com/), you can run `npm install -g babel`. `babel` has built-in support for React v0.12 and v0.13. Tags are automatically transformed to their equivalent `React.createElement(...)`, `displayName` is automatically inferred and added to all React.createClass calls. It's completely compatible with the original ([though deprecated](https://facebook.github.io/react/blog/2015/06/12/deprecating-jstransform-and-react-tools.html)) JSX/React transformer even down to whitespace handling.
```

Comment:
Let's just drop the last sentence here – no one will remember react-tools in a few months anyway.